pr_testing/run-pr-relvals.sh

```diff
@@ -22,7 +22,7 @@ echo "${MATRIX_ARGS}"  | tr ';' '\n' | while IFS= read -r args; do
       high_stats ) ;;
       nano ) OPTS="-w nano" ;;
       * ) if is_in_array "${TEST_FLAVOR}" "${ALL_GPU_TYPES[@]}" ; then
-            OPTS="-w gpu"
+            OPTS="-w gpu --gpu required"
```

Comment:
@iarspider , as this option is only added via https://github.com/cms-sw/cmssw/pull/47377 and will be available in 15.1.X, so we should only add it if runthematrix supports it otherwise it will fail for old release cycles

Reply:
@iarspider , better to add a function in `_helper_functions.sh` and call it every where it is needed. `_helper_functions.sh` is already sourced via `setup-pr-test-env.sh` in this script. For `run-ib-pr-matrix.sh`  you can just add the source statement at start